lib/bolt/node/winrm.rb

```diff
@@ -462,11 +466,11 @@ def make_tempdir
     end
 
     def with_remote_file(file)
-      ext = File.extname(file)
-      ext = VALID_EXTENSIONS.include?(ext) ? ext : '.ps1'
-      file_base = File.basename(file, '.*')
+      file_base = File.basename(file)
+      # Default to powershell if no extension present
```

Comment:
Does this add value? Is there a real chance we'll get a powershell script without a .ps1 extension?

Reply:
I'm not sure, it was there originally. Our integration tests rely on this behavior, but could be updated not to.

Reply:
I would hope you don't... also seems better to be explicit in the case where you've written code that's not PowerShell, it confuses the PS interpreter, and accidentally does bad things.